recipe/meta.yaml

```diff
@@ -57,14 +57,15 @@ test:
     - examples/demo.py
   commands:
     - echo %OCL_ICD_FILENAMES%  # [win]
-    - python examples/demo.py
+    # The next test crashes in emulated ppc64le:
+    - python examples/demo.py  # [not ppc64le]
```

Comment:
How confident are we that the compilation actually results in a package that works on actual hardware?

Reply:
I think we are reasonable confident that it will work, but I can test the package manually as soon as it is uploaded. We are also not testing MacOS/arm64, and so far it seems to have been working fine.

Reply:
Alright, let's give it a shot. :)